mne_sandbox/__init__.py

```diff
@@ -1 +1,2 @@
 from . import preprocessing
+from . import cfc
```

Comment:
should probably be in `mne_sandbox/connectivity/cfc` -- try to mirror `mne` structure where possible